client/constants/messages.js

```diff
@@ -45,12 +45,24 @@ const messages = {
   FETCH_SYNC_RECORDS: _, /* @param Array.<string> categoryNames, @param {number} startAt (in seconds) */
   /**
    * webview -> browser
-   * browser must update its local values with the new sync records, performing
-   * conflict-resolution as necessary.
+   * browser should find objects with matching objectIds then call
+   * RESOLVE_SYNC_RECORDS to perform conflict resolution.
    */
   RECEIVE_SYNC_RECORDS: _, /* @param {string} categoryName, @param {Array.<Object>} records */
```

Comment:
correct, those two calls would only call with same-category records.
browser clients can examine each record's objectData to determine what to do with them.